chromium/pages/cancel/ux.js

```diff
@@ -20,9 +20,23 @@ function replaceLink(explainer) {
   observer.disconnect();
   const linkText = chrome.i18n.getMessage("cancel_he_blocking_network");
   const link = document.createElement("a");
+  link.classList.add("wikilink");
   link.href = "https://en.wikipedia.org/wiki/Downgrade_attack";
   link.innerText = linkText;
   explainer.innerHTML = explainer.innerHTML.replace(linkText, link.outerHTML);
+
+  /*
+    In response to translation of i18n string "cancel_he_blocking_network".
+    Within context of the paragraph and as a standalone string can be interpreted differentlty
+    langauge to language.
+
+    So if link fails to swap in replace, this conditional is triggered
+  */
+  if(document.getElementsByClassName("wikilink").length === 0) {
```

Comment:
```suggestion
  if (document.getElementsByClassName("wikilink").length === 0) {
```

Also you may do this instead:
```suggestion
  if (!document.querySelector('.wikilink')) {
```